src/three/plugins/LoadRegionPlugin.js

```diff
@@ -55,10 +56,17 @@ export class LoadRegionPlugin {
 		const boundingVolume = tile.cached.boundingVolume;
 		const { regions, tiles } = this;
 
+		let mask = false;
 		let visible = false;
 		let maxError = - Infinity;
 		for ( const region of regions ) {
 
+			if ( region.mask ) {
+
+				mask = true;
+
+			}
+
```

Comment:
I'm not sure about this logic. It looks like if any region is specified as a "mask" then any region is used as such, which doesn't seem right.